### PR TITLE
Feature: Add a pre filled input when default month and year are defined

### PR DESCRIPTION
**Pull request by thiagovictorino**
- Update the README file
- Share the method monthsByLang on mixin
- Fix some indentation problems

### CODE REVIEW

File: src/MonthPickerInput.vue [hunk 1]
```diff
@@ -1,78 +1,83 @@
 <template>
   <div
-    class="month-picker-input-container"
-    v-click-outside="hide"
+          class="month-picker-input-container"
```
Comment (kleinrein) [inline src/MonthPickerInput.vue]: The indent looks a bit heavy here. Could it be decreased?

File: src/MonthPickerInput.vue [hunk 2]
```diff
@@ -1,78 +1,83 @@
...
     >
     </month-picker>
   </div>
 </template>
 
 <script>
-import Vue from 'vue'
-import MonthPicker from './MonthPicker.vue'
-import monthPicker from './month-picker'
+  import Vue from 'vue'
```
Comment (kleinrein) [inline src/MonthPickerInput.vue]: No need to indent the script. I'm gonna add ESLint so it's easier to contribute.

File: src/MonthPickerInput.vue (hunk 1 above)
Comment (thiagovictorino) [reply src/MonthPickerInput.vue]: Sure, I will make these changes

**Comment by thiagovictorino** [comment]
I think that everything is ok now, let me know if you think that something have to be done.

File: src/MonthPickerInput.vue (hunk 2 above)
Comment (thiagovictorino) [reply src/MonthPickerInput.vue]: I leave the indentation where I think that is reasonable.

**Comment by thiagovictorino** [comment]
Cool! It was a pleasure to help!

**Pull request merged**